### PR TITLE
[#10101] Reintroduce account generation on keycard multiaccount

### LINKED ISSUE #10101: Add support for adding new accounts in wallet with keycard
# Feature Issue

Add support for adding new accounts in a wallet with keycard

After we implemented private key and seed support, and reworked adding accounts feature https://github.com/status-im/status-react/pull/10100  we need to add keycard support

### PR DESCRIPTION
**Pull request by rasom**
fix #10101 

status: ready

### CODE REVIEW

File: src/status_im/ui/components/bottom_sheet/events.cljs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 (ns status-im.ui.components.bottom-sheet.events
   (:require [status-im.utils.fx :as fx]
-            [status-im.utils.handlers :as handlers]))
+            [status-im.utils.handlers :as handlers]
+            [taoensso.timbre :as log]))
```
Comment (flexsurfer) [inline src/status_im/ui/components/bottom_sheet/events.cljs]: not  used?

File: src/status_im/ui/screens/wallet/add_new/views.cljs [hunk 2]
```diff
@@ -126,39 +127,48 @@
...
+        :on-press
+        #(re-frame/dispatch [:hardwallet/new-account-pin-sheet
+                             {:view {:content pin
+                                     :height  256}}])
+        :disabled?
+        (or add-account-disabled?
+            (and
+             (not (= type :watch))
+             (and
+              (not keyword?)
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/add_new/views.cljs]: `not keyword?` ?

File: src/status_im/ui/screens/wallet/add_new/views.cljs [hunk 3]
```diff
@@ -126,39 +127,48 @@
...
         :label               :t/add-account
         :accessibility-label :add-account-add-account-button
-        :on-press            #(re-frame/dispatch [:wallet.accounts/add-new-account
-                                                  (ethereum/sha3 @entered-password)])
-        :disabled?           (or add-account-disabled?
-                                 (and
-                                  (not (= type :watch))
-                                  (not (spec/valid? ::multiaccounts.db/password @entered-password))))}}]]))
+        :on-press
+        #(re-frame/dispatch [:hardwallet/new-account-pin-sheet
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/add_new/views.cljs]: what about regular account?

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/wallet/add_new/views.cljs (hunk 2 above)
Comment (rasom) [reply src/status_im/ui/screens/wallet/add_new/views.cljs]: shit happens

File: src/status_im/ui/screens/wallet/add_new/views.cljs (hunk 3 above)
Comment (rasom) [reply src/status_im/ui/screens/wallet/add_new/views.cljs]: Yep right, I just thought of merging this pr and creating an issue like #10101 for a regular multiacc. Will fix soon

Comment (rasom) [reply src/status_im/ui/screens/wallet/add_new/views.cljs]: fixed

File: src/status_im/ui/screens/wallet/add_new/views.cljs (hunk 2 above)
Comment (rasom) [reply src/status_im/ui/screens/wallet/add_new/views.cljs]: fixed

File: src/status_im/ui/components/bottom_sheet/events.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/components/bottom_sheet/events.cljs]: fixed

**Review by vitvly — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by flexsurfer** [comment]
@churik @Serhy for me this issue looks important to be manual tested

**Comment by churik** [comment]
Probably yes, but I think it's up to dev, as we mentioned when discussing process. now we are monitoring PRs with `request-manual-qa` label.

**Pull request merged**